docs/Tags.md

```diff
@@ -25,43 +25,51 @@ FT.CREATE ... SCHEMA ... {field_name} TAG [SEPARATOR {sep}]
 SEPARATOR defaults to a comma (`,`), and can be any printable ASCII character. For example:
 
 ```
-FT.CREATE idx SCHEMA tags TAG SEPARATOR ";"
+FT.CREATE idx ON HASH PREFIX 1 test: SCHEMA tags TAG SEPARATOR ";"
```

Comment:
Index examples that lack `ON HASH PREFIX ...` implicitly depend on the deprecated FT.ADD command. We should always show FT.CREATE with the new ON HASH... options.